fil-proofs-tooling/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 storage-proofs-core = { path = "../storage-proofs-core", version = "^10.0.0", default-features = false}
 storage-proofs-porep = { path = "../storage-proofs-porep", version = "^10.0.0", default-features = false }
 storage-proofs-post = { path = "../storage-proofs-post", version = "^10.0.0", default-features = false }
+storage-proofs-update = { path = "../storage-proofs-update", version = "^10.0.0", default-features = false }
```

Comment:
`storage-proofs-update/cuda` and `storage-proofs-update/opencl` should be include in the features below.

Reply:
Thanks, thought I had them all previously, but didn't.